ui/components/Link.tsx

```diff
@@ -82,4 +82,11 @@ function Link({
 
 export default styled(Link)`
   text-decoration: none;
+
+  &.ellipsis {
+    max-width: 300px;
```

Comment:
Why would it always be 300px? This seems like something we should just override with CSS where we need to.